telco-examples/mgmt-cluster/airgap/eib/kubernetes/helm/values/rancher.yaml

```diff
@@ -1,4 +1,4 @@
-hostname: rancher-${MGMT_CLUSTER_IP}.sslip.io
+hostname: ${MGMT_CLUSTER_IP}
```

Comment:
If we change this here, it should also be changed in the other references in the repository.

Reply:
no, this is only because it has to be reached by IP instead of DNS (air gap scenario). Without networking slip.io cannot be reached. That's why I changed (and tested) the IP working fine